### PR TITLE
Add CrowdStrike Falcon Operator policy

### PR DESCRIPTION
**Pull request by redhatrises**
- This adds the capability to deploy the CrowdStrike Falcon Operator via policy

Signed-off-by: Gabe Alford <redhatrises@gmail.com>

### CODE REVIEW

File: community/SI-System-and-Information-Integrity/policy-crowdstrike-falcon-rhmp.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# This policy installs the CrowdStrike Falcon Operator.  For more information about
+# CrowdStrike, see https://crowdstrike.com/
+#
+# This policy is set to enforce by default. If necessary, update the placement rule.
```
Comment (gparvin) [inline community/SI-System-and-Information-Integrity/policy-crowdstrike-falcon-rhmp.yaml]: Can we make it inform by default?  That's our general guidance -- start out of the box with inform.

File: community/SI-System-and-Information-Integrity/policy-crowdstrike-falcon-rhmp.yaml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-crowdstrike-falcon-rhmp
+  annotations:
+    policy.open-cluster-management.io/standards: "NIST SP 800-53, NIST SP 800-171, ISO/IEC 27001"
+    policy.open-cluster-management.io/categories: "AU - AUDIT AND ACCOUNTABILITY, CA - ASSESSMENT AUTHORIZATION AND MONITORING, IA - IDENTIFICATION AND AUTHENTICATION, IR - INCIDENT RESPONSE, SI - SYSTEM AND INFORMATION INTEGRITY"
+    policy.open-cluster-management.io/controls: "AU-2, AU-3, AU-4, AU-5, AU-6, AU-9, AU-12, AU-14, AU-15, CA-7, IA-10, IR-4, IR-5, SI-3, SI-4, 3.3.1, 3.3.2, 3.3.4, 3.3.8, 3.6.1, 3.6.2, 3.14.2, 3.14.3, 3.14.4, 3.14.5, 3.14.6, A.12.4.1, A.16.1.4, A.16.1.5, A.12.2.1"
+spec:
+  remediationAction: enforce
```
Comment (gparvin) [inline community/SI-System-and-Information-Integrity/policy-crowdstrike-falcon-rhmp.yaml]: Only change this remediationAction to inform by default.  The others are ok.  Thanks

File: community/SI-System-and-Information-Integrity/policy-crowdstrike-falcon-rhmp.yaml (hunk 1 above)
Comment (redhatrises) [reply community/SI-System-and-Information-Integrity/policy-crowdstrike-falcon-rhmp.yaml]: @gparvin done. Thanks!

File: community/SI-System-and-Information-Integrity/policy-crowdstrike-falcon-rhmp.yaml (hunk 2 above)
Comment (redhatrises) [reply community/SI-System-and-Information-Integrity/policy-crowdstrike-falcon-rhmp.yaml]: This is done as well

**Review by gparvin — APPROVED** [review]
Thanks for adding to the policy collection!

**Pull request merged**